packages/dcos-integration-test/extra/conftest.py

```diff
@@ -431,16 +432,26 @@ def _destroy_complete(deployment_id):
                         "completed in {} seconds.".format(timeout))
 
 
-@pytest.fixture(scope='module')
-def registry_cluster(cluster, request):
+@pytest.yield_fixture(scope='module')
```

Comment:
Did you confirm that to work? When using a `yield_fixture`, there should also be the `yield` keyword in the body.